Harvest.Web/Controllers/AppleController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Harvest.Core.Data;
+using Harvest.Core.Domain;
+using Harvest.Web.Models.Settings;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Harvest.Web.Controllers
+{
+    public class AppleController : Controller
+    {
+        private readonly AppDbContext _dbContext;
+        private readonly AuthSettings _authSettings;
+
+        public AppleController(AppDbContext dbContext, IOptions<AuthSettings> authSettings)
```

Comment:
maybe call it `AppleReviewController` or `MobileReviewController` because i think i'll forget what an AppleController is after a while